project-metrics/metrics_service/database/commit_scraper.py

```diff
@@ -104,7 +104,6 @@ def scrape_since_latest(self):
         db.get_engine().execute(
             models.Commit.__table__.insert().prefix_with('IGNORE'),
             commit_dicts)
-        self.session.commit()
```

Comment:
?

Reply:
Commit message was "Remove unnecessary session.commit()". I wasn't using the session to execute, I was using the engine directly; this was an artifact from an alternative approach.